src/components/SocialSharingButtons.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div class="social-container">
+    <h1>{{ $i18n.t("home.share") }}</h1>
```

Comment:
Finally I think this text is no longer necessary.  
Maybe you can delete this line and the associated translations 😉